server/src/test/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActorSpec.scala

```diff
@@ -52,6 +54,8 @@ class MaterializeWorkflowDescriptorActorSpec extends CromwellTestKitWordSpec wit
   val workflowSourceNoDocker = HelloWorld.workflowSource(""" runtime { } """)
   val Timeout = 10.second.dilated
   val NoBehaviorActor = system.actorOf(Props.empty)
+  val errorOrCallCachingEnabled = validate(true)
```

Comment:
Is there anywhere negative scenario test?

Reply:
UPD: this variables can be only true of false. Only its presence is important here. Behavior of the method, taking this vals as arguments, testing at special test ValidatingCallCachingModeSpec

Reply:
So are there tests where `val errorOrCallCachingEnabled = validate(false)` ?

I mean I haven't seen it in this mr.